### ISSUE TITLE
Run tests fails in Babashka REPL

### ISSUE THREAD

**Issue by katox** [cursive-ide/cursive#2999]
Running tests (or one test) in bb remote repl fails with an exception:
```
Running template-test/nah
java.lang.IllegalArgumentException: null object for public java.lang.String java.lang.StackTraceElement.getClassName() user-f590c0fc-2d13-4bbc-a01f-8fb94c6f59a7 $HOME/.local/share/JetBrains/IntelliJIdea2024.3/clojure-plugin/lib/runtime.jar!/cursive/tests/report.clj:50:21
```

As a workaround - if the test doesn't contain any bb specific functionality - it is still possible to run a test as a java process using the green triangle next to the test.

**Comment by dconlin-gresham** [comment]
Yes, I have this problem too. It seems to be strictly linked to the "run tests" functionality in Cursive, since I can work around it by:
- working out if I have any failing tests by using `clojure.test/run-tests` and `clojure.test/run-all-tests`
- debugging known failing tests by running the forms inside the test in the repl.

Of course, this is a lot less convenient than being able to run and debug tests using the existing cursive ui and shortcuts though!

I'm running on Mac - I'm not sure if that helps at all with debugging.
